micro_brewery/data/product_template_attribute_line.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_template_attribute_line_2" model="product.template.attribute.line" context="{'create_product_product': False}">
```

Comment:
It is useful because if we want to add different attributes to same product then this context is required else it will throw error that this template already exists.